Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -6324,10 +6324,9 @@ def pascalToSpace(s):
     if not isinstance(s, STRING_OBJ_TYPES):
         return s
 
-    tokens = pascalRegex.findall(s)
-    for t in tokens:
-        # double space to handle capital words like IP/URL/DNS that not included in the regex
-        s = s.replace(t, ' {} '.format(t.title()))
+    s = s[0].upper() + s[1:]
```

Comment:
Why not use capitalize?

Reply:
Because the string is given as one concatenated word and using `capitalize` will lead to lowering all chars other than the first one.
Anyway, I removed it and changed to use `title` inside `re.sub`, to catch another edge case.